bin/makeSimRun

```diff
@@ -1,51 +1,46 @@
-    makeSim --detector $detector --pfiDesignId 1 --exptime 0 --noSky --expId 7 --imagetyp BIAS
-    makeSim --detector $detector --pfiDesignId 1 --exptime 0 --noSky --expId 8 --imagetyp BIAS
-    makeSim --detector $detector --pfiDesignId 1 --exptime 0 --noSky --expId 9 --imagetyp BIAS
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 10 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 11 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 12 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 13 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 14 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 15 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 16 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 17 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 18 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 1 --exptime 900 --noSky --expId 19 --imagetyp DARK
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 20 --imagetyp FLAT --xoffset 0 --detectorMap detectorMap-sim-${detector}.fits
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 21 --imagetyp FLAT --xoffset 1000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 22 --imagetyp FLAT --xoffset 2000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 23 --imagetyp FLAT --xoffset 3000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 24 --imagetyp FLAT --xoffset 4000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 25 --imagetyp FLAT --xoffset 5000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 26 --imagetyp FLAT --xoffset -1000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 27 --imagetyp FLAT --xoffset -2000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 28 --imagetyp FLAT --xoffset -3000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 29 --imagetyp FLAT --xoffset -4000
-    makeSim --detector $detector --pfiDesignId 2 --exptime 30 --noSky --expId 30 --imagetyp FLAT --xoffset -5000
-    makeSim --detector $detector --pfiDesignId 3 --exptime 1 --noSky --expId 31 --imagetyp ARC
-    makeSim --detector $detector --pfiDesignId 4 --exptime 900 --expId 32 --imagetyp OBJECT --allOutput
-    makeSim --detector $detector --pfiDesignId 4 --exptime 900 --expId 33 --imagetyp OBJECT --allOutput
+    makeSim --detector $detector --pfiDesignId $pfiDesignId --exptime 0 --domeClosed --expId 0 --imagetyp BIAS
```

Comment:
Is this change relevant for this particular ticket?

Reply:
I think it is not unreasonable: this ticket needs to clarify how to specify the source of the light going into the fibers. Making it harder to get "sky" along with your cals seems like a good thing.

But I'd ask whether it would be clearer to directly use the --imagetyp ?

Reply:
Because the PfsConfigs are no longer dark/flat/science specific, we only need two instead of the four we had previously. This change is why the ticket was done in the first place.

@CraigLoomis is right about tying things to `--imagetyp`; I have always viewed that as just manipulating the header, but it should be smarter.